### PR TITLE
Review "Getting Started" Docs

### PR DESCRIPTION
**Pull request by retiutut**

### CODE REVIEW

**Comment by daniellasry** [comment]
### In the Cyton getting started guide:
- The "Forum" link leads to a 404 page. I think it needs an extra slash at the end of the link:
> If you are still having troubles connecting to your Cyton board, refer to the **Forum** for extra troubleshooting advice.

- Something wrong with the Community Page link:
image

- The entire section, "TROUBLE SHOOTING ON MACOS SIERRA" can be deleted, IMO

**Comment by daniellasry** [comment]
### In the Ganglion getting started guide:
- This "troubleshooting" link doesn't do anything:
image

- The section "RUN THE GUI ON MACOS SIERRA" can be deleted

**Comment by retiutut** [comment]
Attack of the dead links.... at the computer now making the changes

**Comment by daniellasry** [comment]
### Wifi Getting Started Guide
- Push The World link is broken, maybe make it not a link?
image

- These links go to specific version of the firmware. Instead, we should link to the "latest" version like this:
https://github.com/OpenBCI/OpenBCI_32bit_Library/releases/latest
https://github.com/OpenBCI/OpenBCI_Ganglion_Library/releases/latest
image

- Maybe this firmware link should lead to https://github.com/OpenBCI/OpenBCI_32bit_Library/releases
image

- This link should lead to: https://github.com/OpenBCI/OpenBCI_WIFI/releases/latest
image

- Same with this "latest firmware" link at the bottom of the page:
image

File: Tutorials/01-Cyton_Getting Started_Guide.md [hunk 1]
```diff
@@ -452,28 +452,6 @@ You'll notice that the strongest alpha wave signals should be appearing in chann
 
 For more ideas on what to do next, check out the OpenBCI [Community Page] (http://openbci.com/community/) and the other OpenBCI Docs pages.
```
Comment (daniellasry) [inline Tutorials/01-Cyton_Getting Started_Guide.md]: I think the space after [Community Page] needs to be removed

Comment (daniellasry) [reply Tutorials/01-Cyton_Getting Started_Guide.md]: Everything else looks good. Thanks richard!

Comment (retiutut) [reply Tutorials/01-Cyton_Getting Started_Guide.md]: This PR includes the general flow, readability, and functionality of the most used documents. There may be more drastic changes to the Ganglion docs that reflects the current state of the hardware, and also updated screenshots. Maybe better to contain that in another PR.

**Comment by daniellasry** [comment]
Did you forget to pull from upstream before making changes? I'm seeing some outdated stuff in the diff, and you have some merge conflicts.

**Comment by retiutut** [comment]
Yes I did forget to pull upstream, how do I fix it? 😞

**Comment by daniellasry** [comment]
this should work:
```
> git fetch upstream
> git merge upstream/master
```

Then, it will ask you to clear out the merge conflicts in a text editor.

Alternatively you can hit the "Resolve conflicts" button below. If you do that, make sure you run this afterwards:
```
> git fetch origin
> git merge origin/master
```

**Comment by retiutut** [comment]
@daniellasry Thanks!

**Pull request merged**